### ISSUE TITLE
Please add more clear install instuctions

### ISSUE THREAD

**Issue by clankill3r** [zaphoyd/websocketpp#1057]
It would be really nice if the readme.md had a bit more instructions of how to use this library.
Mention that boost is required. Which version.
What commands to run etc.

Now I have been just looking threw different issues posted by users to get started and i'm stuck on this now https://stackoverflow.com/questions/73430102/undefined-symbols-for-architecture-x86-64-boostthis-threadinterruption-poi

and at this point i'm drained instead of exited to learn about this library.

**Comment by rvkennedy** [comment]
Yes, specifically which asio version to use would be a start: both the deployed Asio 1.24 and the current github master cause compiler errors with websocketpp for me.
